torch/sparse/__init__.py

```diff
@@ -61,41 +61,63 @@
 .. note::
     This function doesn't support computing derivaties with respect to CSR matrices.
 
-    Args:
-        mat1 (Tensor): the first sparse matrix to be multiplied
-        mat2 (Tensor): the second matrix to be multiplied, which could be sparse or dense
-
-    Shape:
-        The format of the output tensor of this function follows:
-        - sparse x sparse -> sparse
-        - sparse x dense -> dense
-
-    Example::
+    This function also additionally accepts an optional :attr:`reduce` argument that allows
+    specification of an optional reduction operation, implemented only for CSR storage format
+    on CPU device.
```

Comment:
sure, but i am not very familiar with how to write this latex, can you help here ?

Reply:
I guess this would look something like
```
.. math::

    z_{ij} = \bigoplus_{k = 1}^K x_{ik} y_{kj}

where :math:`bigoplus` defines the reduce operator.
```

Reply:
Thanks!